internal/trelloprovider/model.go

```diff
@@ -0,0 +1,69 @@
+package trelloprovider
+
+import (
+	"fmt"
+	"github.com/cayleygraph/quad"
+	"github.com/adlio/trello"
+	"moul.io/depviz/v3/internal/dvmodel"
+	"moul.io/depviz/v3/internal/dvparser"
+)
+
+func fromCards(cards []*trello.Card) dvmodel.Batch {
+	batch := dvmodel.Batch{}
+	for _, issue := range cards {
+		err := fromCard(&batch, issue)
+		if err != nil {
+			fmt.Println("error")
+			continue
+		}
+	}
+	return batch
+}
+
+func fromCard(batch *dvmodel.Batch, input *trello.Card) error {
+	
+	entity, err := dvparser.ParseTarget(input.URL)
+	if err != nil {
+		return fmt.Errorf("parse target: %w", err)
+	}
+
+	card := dvmodel.Task{
+		ID:           quad.IRI(entity.String()),
```

Comment:
I'm not sure that `entity.String` is always unique. Maybe we can point to the same trello item with two different URLs.

Can you check in the `input` if there is no official URL instead maybe?